packages/next/src/lib/server-external-packages.json

```
@@ -23,6 +23,7 @@
   "postcss",
   "prettier",
   "prisma",
+  "puppeteer",
```

```suggestion
  "playwright",
  "postcss",
  "prettier",
  "prisma",
  "puppeteer",
```

Is it possible to do the same with [`playwright`](https://www.npmjs.com/package/playwright)? I have to use a [trick with `eval()`](https://github.com/kachkaev/website/blob/c3a149befaf26835e0c13991fb1a344e2e189059/app/%5Blocale%5D/update-profiles/shared/handler-helpers.ts#L35-L38) to make it work inside a Next.js route handler. It’s a cron job.

---

Done by @jocarrd in https://github.com/vercel/next.js/pull/49649, thanks!